### PR TITLE
DCA-250 Create statistic collector script

### PR DESCRIPTION
**Pull request by SergeyMoroz0703**
Python script `statistic_collector.py` will be run after each `bzt` execution in case if `allow_analytics` enabled (by default) in .yml configuration files.
It  will not collect any PI data.

### CODE REVIEW

File: app/util/project_paths.py [hunk 1]
```diff
@@ -5,6 +5,13 @@ def __get_jira_yml():
     return Path(__file__).parents[1] / "jira.yml"
 
 
+def __get_jira_result_dir():
+    return Path(__file__).parents[1] / "results" / "jira"
```
Comment (astashys) [inline app/util/project_paths.py]: you can make 1 method name it for example `get_artifacts_dir` and `use return os.environ.get('TAURUS_ARTIFACTS_DIR')` Also reuse it in conftest.py script

File: app/util/analytics.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+# List in value in case of specific output appears for some OS for command platform.system()
+OS = {'macOS': ['Darwin'], 'Windows': ['Windows'], 'Linux': ['Linux']}
+DT_REGEX = r'(\d{4}-\d{1,2}-\d{1,2}\s+\d{1,2}:\d{1,2}:\d{1,2})'
+
+BASE_URL = 'http://dcapps-ua-test.s3.us-east-2.amazonaws.com/stats.html?'
+DEV_BASE_URL = 'http://dcapps-ua-test.s3.us-east-2.amazonaws.com/stats_dev.html?'
+
+
+def __validate_app_type():
+    msg = 'Please run util/analytics.py with application type as argument. E.g. python util/analytics.py jira'
```
Comment (astashys) [inline app/util/analytics.py]: can be constant on top level

File: app/util/analytics.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+                      f'exp_dur={self.run_analytics.duration}&act_dur={self.run_analytics.actual_duration}&' \
+                      f'sel_count={self.run_analytics.selenium_test_count}&jm_count={self.run_analytics.jmeter_test_count}&' \
+                      f'concurrency={self.run_analytics.concurrency}'
+
+        r = requests.get(url=f'{base_url}{params_string}')
+
+
+if __name__ == '__main__':
+    app_type = application_type()
+    p = AnalyticsFormer(app_type)
```
Comment (astashys) [inline app/util/analytics.py]: maybe we can change the name `p` to something more understandable?

File: app/util/analytics.py [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    except IndexError:
+        SystemExit(msg)
+
+
+def application_type():
+    __validate_app_type()
+    return sys.argv[1]
+
+
+class AnalyticsFormer:
```
Comment (astashys) [inline app/util/analytics.py]: What do you think about `AnalyticsCollector`?

File: app/util/analytics.py [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+def __validate_app_type():
+    try:
+        app_type = sys.argv[1]
+        if app_type.lower() not in [JIRA, CONFLUENCE, BITBUCKET]:
+            raise SystemExit(APP_TYPE_MSG)
+    except IndexError:
+        SystemExit(APP_TYPE_MSG)
+
+
+def application_type():
```
Comment (ometelytsia) [inline app/util/analytics.py]: let's rename to `get_application_type`

**Comment by astashys** [comment]
Did you try negative cases, for example, if you provided an inexistent host or when Jira server in captcha mode?

**Pull request merged**